lib/board.rb

```diff
@@ -0,0 +1,62 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    self.cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+    @cells
+  end
+
+def display
+    puts " "
+    puts "     Here is the current board:"
+    puts " "
+    puts "      #{self.cells[0]} | #{self.cells[1]} | #{self.cells[2]} "
+    puts "     -----------"
+    puts "      #{self.cells[3]} | #{self.cells[4]} | #{self.cells[5]} "
+    puts "     -----------"
+    puts "      #{self.cells[6]} | #{self.cells[7]} | #{self.cells[8]} "
+  end
+
+
+  def position(number)
+    self.cells[number.to_i - 1]
+  end
+
+  def full?
+    return true if self.cells.none? { |position| position == " " }
```

Comment:
Avoid returning boolean literals `true` and `false`. Instead, rely on the implicit return value of the boolean expression.
```
def full?
  self.cells.none? { |position| position == " " }
end
```